sdk/keyvault/Azure.Security.Keyvault.Keys/src/RSAKey.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Azure.Security.Keyvault.Keys
+{
+    public class RSAKey
+    {
```

Comment:
I'm not sure that I wan't to create specific classes for RSAKey and ECKey at this point.  Also it seems odd if we had these classes that they wouldn't be derived from key.

Reply:
changed after our conversation on Teams about having `KeyCreateOptions`